code/modules/vore/resizing/resize_vr.dm

```diff
@@ -242,6 +242,19 @@
 		return TRUE
 	return FALSE
 
+/mob/living/proc/can_pass_into(turf/n, var/prey = null)
```

Comment:
Honestly, this proc is better served just inserted into general handle_bump_other than being its own proc, for better readability, imo

Reply:
fair